aws/cloudformation/components/database.yml.erb

```diff
@@ -1,23 +1,24 @@
+<% if database-%>
```

Comment:
I was a little confused why you moved this conditional inside the subcomponent, but I finally figured it out. A comment at the top might make this easier to understand.
```suggestion
# Provision cloud database resources `if database`, and provision connection configuration parameters for both cloud and local database environments.

<% if database-%>
```